README.md

```diff
@@ -190,9 +190,18 @@ To install NumPyro with a CPU version of JAX, you can use pip:
 pip install numpyro
 ```
 
-To use NumPyro on the GPU, you will need to first [install](https://github.com/google/jax#installation) `jax` and `jaxlib` with CUDA support.
+To use NumPyro on the GPU, you need to install CUDA first and then use the following pip command:
+```
+pip install numpyro[cudaXXY] -f https://storage.googleapis.com/jax-releases/jax_releases.html
+```
+where `XXY` is your CUDA XX.Y version number, e.g., for CUDA 10.2 use `cuda[102]` in the above.
+If you need further guidance, please have a look at the [JAX GPU installation instructions](https://github.com/google/jax#pip-installation-gpu-cuda).
 
-To run NumPyro on Cloud TPUs, you can use pip to install NumPyro as above and setup the TPU backend as detailed [here](https://github.com/google/jax/tree/master/cloud_tpu_colabs).
+To run NumPyro on Cloud TPUs, you need to setup the TPU backend as detailed [here](https://github.com/google/jax/tree/master/cloud_tpu_colabs)
+and then install NumPyro using the following pip command:
+```
+pip install numpyro[tpu] -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
+```
```

Comment:
I think this command can be confusing. If I understand correctly, those people who want to run `pip install numpyro[tpu] ...` are using Cloud TPU VM (not Colab) and they want to skip [this step](https://cloud.google.com/tpu/docs/jax-quickstart-tpu-vm) in the Cloud TPU VM JAX setup process. I'm not sure we should encourage such simplification. In my opinion, we should encourage users to follow the upstream quickstart, test the environment, then install numpyro with `pip install numpyro`.

Reply:
Alright, I'll revisit this.

Reply:
Alright, maybe this is fine now?